src/App.jsx

```diff
@@ -9,14 +9,27 @@ import LoginForm from "./components/LoginForm";
 import SignUpForm from "./components/SignUpForm";
 import { useDispatch } from "react-redux";
 import { getPlace } from "./modules/location";
+import auth from "./modules/auth";
 
 const App = (props) => {
   const [uid, setUid] = useState("");
   const [authenticated, setAuthenticated] = useState(false);
   const dispatch = useDispatch();
 
-  useEffect(() => {
+  useEffect(async () => {
```

Comment:
Don't use async useEffect, factor out the login function to a separate async function, and call on it from within useEffect

Reply:
^ I agree with this

Reply:
did you forget to remove it from here?

Reply:
No did not forget it wouldnt work, but removed now